### PR TITLE
Create an spf policy

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7531996067/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-10f7dc195c/how-to/). :bow:

File: content/rackspace-email/create-an-spf-policy.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- **Applies to:** Administrator
+- **Difficulty:** Easy
+- **Time needed:** Approximately 15 minutes to create record, 24-48 hours for the record to propagate
+- **Tools required:**  DNS host administrator access
+
+You also need access to update DNS entries for your domain. If you do not know where your DNS is hosted, see [Find your DNS host](/how-to/find-dns-host).
+
+For more information on prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
+
+An **SPF** policy on it's own does not go far in terms of protecting the integrity of your domain's email. In order to create a secure email verification policy, we recommend the following:
```
Comment (stephamon) [inline content/rackspace-email/create-an-spf-policy.md]: An **SPF** policy alone is not sufficient to protect the integrity of your domain's email.

File: content/rackspace-email/create-an-spf-policy.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+1. Log in to the control panel for your domain DNS host.
+
+2. Create a TXT record with the following specifications.
+
+    | Type | Hostname | Destination | TTL |
+    | :---: | :---: | :---: | :---: |
+    | TXT | @ | **v=spf1 include:emailsrvr.com ~all** | 3600 |
+
+3. Decide how you would like to enforce SPF failures.
```
Comment (stephamon) [inline content/rackspace-email/create-an-spf-policy.md]: Decide how you want to enforce SPF failures.

File: content/rackspace-email/create-an-spf-policy.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    - `~all` will result in a soft fail(Not authorized, but not explicitly unauthorized).
```
Comment (stephamon) [inline content/rackspace-email/create-an-spf-policy.md]: `~all` results in a soft fail (Not authorized, but not explicitly unauthorized).

(Make sure to put a space between fail and the parenthetical)

File: content/rackspace-email/create-an-spf-policy.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    - `-all` will result in a hard fail(Unauthorized).
```
Comment (stephamon) [inline content/rackspace-email/create-an-spf-policy.md]: `-all` results in a hard fail (Unauthorized).

(Make sure to put a space between fail and the parenthetical)

File: content/rackspace-email/create-an-spf-policy.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    - `?all` is neutral(As if there is no policy at all).
```
Comment (stephamon) [inline content/rackspace-email/create-an-spf-policy.md]: Make sure to put a space between neutral and the parenthetical

File: content/rackspace-email/create-an-spf-policy.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+- **Time needed:** Approximately 15 minutes to create record, 24-48 hours for the record to propagate
+- **Tools required:**  DNS host administrator access
+
+You also need access to update DNS entries for your domain. If you do not know where your DNS is hosted, see [Find your DNS host](/how-to/find-dns-host).
+
+For more information on prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
+
+An **SPF** policy on it's own does not go far in terms of protecting the integrity of your domain's email. In order to create a secure email verification policy, we recommend the following: 
+
+1. Create an **SPF** record policy.
```
Comment (stephamon) [inline content/rackspace-email/create-an-spf-policy.md]: Should SPF be bold? If so, make sure to bold it consistently throughout. Use the same styling for TXT and other record types.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-6fa9478c69/how-to/). :bow:

File: content/rackspace-email/create-an-spf-policy.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+4. Authorize additional mail servers by adding their IP or server name after **include:**.
+
+    - For example:
+
+    | Type | Hostname | Destination | TTL |
+    | :---: | :---: | :---: | :---: |
+    | TXT| @ | **v=spf1 include:emailsrvr.com include:othermailer.com ~all** | 3600 |
+
+3. Save your changes.
```
Comment (cyrichardson) [inline content/rackspace-email/create-an-spf-policy.md]: 3 ->5

**Pull request merged**